modules/lib/maintainers.nix

```
@@ -223,4 +223,9 @@
     github = "kmaasrud";
     githubId = 54394333;
   };
+    loicreynier = {
+    name = "Loïc Reynier";
+    email = "loic@loicreynier.fr";
+    github = "loicreynier";
+    githubId = 88983487;
```

```suggestion
  loicreynier = {
    name = "Loïc Reynier";
    email = "loic@loicreynier.fr";
    github = "loicreynier";
    githubId = 88983487;
  };
```

There seems to be a missing closing curly brace.